substrate/frame/treasury/src/benchmarking.rs

```diff
@@ -78,8 +78,7 @@ fn create_approved_proposals<T: Config<I>, I: 'static>(n: u32) -> Result<(), &'s
 		#[allow(deprecated)]
 		Treasury::<T, I>::propose_spend(RawOrigin::Signed(caller).into(), value, lookup)?;
 		let proposal_id = <ProposalCount<T, I>>::get() - 1;
-		#[allow(deprecated)]
```

Comment:
all the changes within the current module backward compatible. they ensure that only `approve_proposal` call be disabled via `ApproveOrigin` parameter, without affecting the rest of the calls.